### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by vinchevskaya**

### CODE REVIEW

File: DEV-3/ConverterOfNumber.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            radix = radixNumber;
+        }
+
+        /// <summary>
+        /// This method converts a decimal number to a new number system.
+        /// </summary>
+        /// <returns>Return a converted number in another number system.</returns>
+        public string СonvertDeciminalNumber()
+        {
+            string resultNamber = "";
```
Comment (IgorKheidorov) [inline DEV-3/ConverterOfNumber.cs]: Grammar error, Number

File: DEV-3/ConverterOfNumber.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+            while (decimalNumber > 0)
+            {
+                rest = decimalNumber % radix;
+                if (rest < 9)
+                {
+                    resultNamber += rest;
+                }
+                else
+                {
```
Comment (IgorKheidorov) [inline DEV-3/ConverterOfNumber.cs]: Maybe much more effective use 'A'-10 as a constant not calculate each time?

File: DEV-3/ConverterOfNumber.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        {
+            string resultNamber = "";
+            int rest;
+            int decimalNumber = ConvertNumber;
+
+            while (decimalNumber > 0)
+            {
+                rest = decimalNumber % radix;
+                if (rest < 9)
+                {
```
Comment (IgorKheidorov) [inline DEV-3/ConverterOfNumber.cs:51]: It seem that it will not work, bug. Please test, '0' is not 0

File: DEV-3/ConverterOfNumber.cs [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+                }
+                decimalNumber /= radix;
+            }
+            return (Сonverse(resultNamber));
+        }
+
+        /// <summary>
+        /// This method returns a value.
+        /// </summary>
+        /// <returns>Converted string.</returns>
```
Comment (IgorKheidorov) [inline DEV-3/ConverterOfNumber.cs]: Convert not converse

File: DEV-3/EntryPoint.cs [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+                int decimalNumber = Int32.Parse(arg[0]);
+                if (decimalNumber < 0)
+                {
+                    throw new Exception("Can't convert negative numbers");
+                }
+                int newRadix = Int32.Parse(arg[1]);
+                if (newRadix < 2 || newRadix > 20)
+                {
+                    throw new Exception("wrong Radix");
+                }
```
Comment (IgorKheidorov) [inline DEV-3/EntryPoint.cs]: Why do you not simply pass decimalNumber and newRadix to method

File: DEV-3/EntryPoint.cs [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+{
+    /// <summary>
+    /// It is entry point of the program and it is print a number in the selected number system.
+    /// </summary>
+    class EntryPoint
+    {
+        static void Main(string[] arg)
+        {
+            try
+            {
```
Comment (IgorKheidorov) [inline DEV-3/EntryPoint.cs]: why do not not check for number of arguments?

File: DEV-3/ConverterOfNumber.cs [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+        /// <summary>
+        /// This method converts a decimal number to a new number system.
+        /// </summary>
+        /// <returns>Return a converted number in another number system.</returns>
+        public string СonvertDeciminalNumber()
+        {
+            string resultNamber = "";
+            int rest;
+            int decimalNumber = ConvertNumber;
```
Comment (IgorKheidorov) [inline DEV-3/ConverterOfNumber.cs]: What about decimalNumber  = 0?

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: DEV-3/ConverterOfNumber.cs [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+    {
+        int ConvertNumber;
+        int radix;
+
+        /// <summary>
+        /// Constructor instance.
+        /// </summary>
+        public ConverterOfNumber(int decimalValues, int radixNumber)
+        {
+            if (decimalValues < 0)
```
Comment (AlexanderSoroka) [inline DEV-3/ConverterOfNumber.cs:19]: it's better to implement custom setter/getter instead all these checks

**Review by AlexanderSoroka — APPROVED** [review]

**Pull request merged**